src/ORM/Behavior/Translate/EavStrategy.php

```diff
@@ -131,6 +131,10 @@ protected function setupAssociations(): void
                 $conditions[$name . '.content !='] = '';
             }
 
+            if ($this->table->associations()->has($name)) {
```

Comment:
These checks are mainly required because in our test suite we remove and re-add the Translate behavior with different configs but removing the behavior doesn't remove the associations created on the source table.